portal/src/pages/search.vue

```diff
@@ -185,6 +183,9 @@ export default {
       this.search.page = 1
       this.$store.dispatch('searchMaterials', Object.assign({}, this.search))
       this.$router.push(this.generateSearchMaterialsQuery(this.search))
+    },
+    getFilterCategories() {
+      return this.materials ? this.materials.filter_categories : []
```

Comment:
(this.materials && this.materials.filter_categories) ? this.materials.filter_categories : []
Dan hoeft de [] in de template niet meer lijkt mij op regel 62

Reply:
Ja, dat dacht ik ook. Blijkt toch niet zo te zijn, omdat het een global store is. Dus `this.materials` kan al gevuld zijn uit een andere pagina (zonder filter_categories) waardoor er toch dingen fout gaan. Hier zou dan een extra check bij moeten of die filter_categories bestaat.

Reply:
Pfff, ok, gotcha